contracts/bid/Makefile

```diff
@@ -3,22 +3,22 @@ help: ## Display this help screen
 		-E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | \
 		awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-# wasm: ## Build the WASM files
-# 	@mkdir -p ./target/verifier-keys/
-# 	@cp -fr ~/.rusk/keys/bid-circuits/**/*.vk ./target/verifier-keys
-# 	@cargo rustc \
-# 		--release \
-# 		--target wasm32-unknown-unknown \
-# 		--no-default-features \
-# 		-- -C link-args=-s
+wasm: ## Build the WASM files
+	@cargo rustc \
+		-vvv \
```

Comment:
I don't think is actually makes sense have the extra verbose for building the WASM file; we might pollute the log for nothing.